# Chip gain
gain: 12
# Internal chip data rate. Must be > than sampling rate
# emg_data_rate: 125 # Hz
# Shimmer sampling rate
sampling_rate: 512 # Hz
# Stream 1 Hz sqare wave with +-1 amplitude test signal instead of acquired
stream_test_signal: False
# Calibrate ADC offset using test signal before data acquisition 
calibrate_test_signal: 3 # Calibration duration in seconds (0 - no calibration)
# Size of the queue for asynchronous publishing on topic
queue_size: 10